eng/scripts/Language-Settings.ps1

```diff
@@ -4,6 +4,8 @@ $packagePattern = "*.zip"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/python-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=python%2F&delimiter=%2F"
 
+$SkipChangeLogVerification.Add("azure-data-nspkg", $true)
```

Comment:
Why can't we add a changelog and fix it here instead of trying to skip it?

Reply:
That's what I initially set out to do but @scbedd says there is a reason it has no version/ changelog file.

Reply:
@lmazuel @scbedd can you one of you guys provide more information on why this package doesn't have a ChangeLog?

Reply:
@weshaggard nspkgs are published exactly once. There is very little functional reason to maintain a changelog for it.

Reply:
I will note that the way that that Dozie implemented it can be skipped individually on the artifact basis. That'll definitely work for this purpose.